ar-module-explainer.md

```diff
@@ -68,6 +68,39 @@ function drawScene() {
 }
 ```
 
+### Drawing UI elements
+
+The AR module supports a variety of device form factors, including "handheld" phone AR as well as "headworn" glasses-style AR. If the application wishes to show UI elements to the user, it needs to be done differently based on the form factor. This information is exposed through `XRSession`'s `interactionSpace` attribute.
```

Comment:
When revisiting this topic after a few months away, I first thought `interactionSpace` was going to give me an actual `XRSpace`, which felt like the system-provided rectangle design we moved away from a year ago.  It took me a bit to realize that this was returning an enum.

We should probably call this **`interactionMode`**, as this attribute/enum is very similar to [`targetRayMode`](https://www.w3.org/TR/webxr/#enumdef-xrtargetraymode) and not similar to `targetRaySpace`.

Reply:
This works for me! I'm not attached to the naming.

Reply:
renamed.